library/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/callback/RecorderWindowCallback.kt

```diff
@@ -123,7 +123,15 @@ internal class RecorderWindowCallback(
         if (pointerInteractions.isEmpty()) {
             return
         }
-        processor.processTouchEventsRecords(ArrayList(pointerInteractions))
+
+        val item = recordedDataQueueHandler.addTouchEventItem(
+            ArrayList(pointerInteractions)
+        ) ?: return
+
+        if (item.isReady()) {
```

Comment:
why do you check here ? I think this value is `true` by default.

Reply:
This was for future proofing. My assumption was that if in future the isReady has any sort of logic it would be better to already wrap this so that we don't forget to do so later on